content/vendor/guides/using-ecr-private-images.md

```diff
@@ -0,0 +1,319 @@
+---
+date: "2020-09-24T00:00:00Z"
+lastmod: "2020-09-24T00:00:00Z"
+title: "Using ECR for Private Images"
+weight: "1005"
+---
+
+KOTS supports working with private images stored in Amazon's Elastic Container Registry (ECR).
+
+## Objective
+
+The purpose of this guide is to walk you through a hello-world example on how to configure Replicated to pull images from a private registry in Amazon's Elastic Container Registry (ECR).
+
+## Prerequisites & Assumptions
+
+This power-user's guide assumes you have completed the [standard quickstart](https://kots.io/vendor/guides/quickstart/) or the [CLI quickstart](https://kots.io/vendor/guides/cli-quickstart/) guides as this guide is a continuation of those guides.
```

Comment:
Is this a power-user's guide or a guide for anyone to complete?

Reply:
I copied/pasted this from another guide so just assume that any guide that goes beyond the quick starts is a power user guide. But if I misread what an power user guide is, I can easily just take 'poweriser's' out.